src/pages/home/Home.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+import { PropTypes } from 'prop-types';
+
+import { indigo, white, darken, green } from '../../styles/colors';
+import { calculateRem } from '../../styles';
+import { welcome, currentTime } from '../../actions';
+
+class Home extends Component {
+  changeTime = () => {
+    const { currentTimeAction, welcomeAction } = this.props;
+    const time = new Date().toLocaleTimeString();
+    welcomeAction();
+    currentTimeAction(time);
+  };
+
+  greet = () => {};
+
+  render() {
+    const { message, time } = this.props;
+    return (
+      <StyledHome>
+        <h2>
+          Welcome To Memeing Maker setup
+          <StyledSpan color={white}>
+            A span that accepts a color prop
+          </StyledSpan>
+        </h2>
+        <h2>
+          {message} {time && `the time is ${time}`}
+        </h2>
+        The container below is a flex box
+        <Container>
+          <div>A Flex element</div>
+          <div>Another Flex element</div>
+        </Container>
+        <p>
+          This is intended to demonstrate some of the ways we can style using
+          styled components and combine it with grid-styled
+        </p>
+        <Button color={green} onClick={this.changeTime}>
+          Change Time
+        </Button>
+      </StyledHome>
+    );
+  }
+}
+
+const StyledHome = styled.div`
```

Comment:
@brainyfarm thanks for this question. Here are some of the reasons I decided to go with styled components instead of separate CSS files:

1. **It Makes Components Cleaner and concerns separate:** It removes the need to clutter components with `className` props and makes use of components that makes it clean and semantically easy to understand. 
2. **It makes components very reusable and customizable:** With the use of props, it is very easy to reuse a component with different stylings. For instance, With a single `Button` component that accepts color, size, bordered, disabled, etc. props, I can create buttons of different colors, sizes with/out borders etc by just passing the appropriate props.
3. **It eliminates the problem of scoping and selector collision:** Styled components use randomly generated class names. this eliminates the problem of collision in styling and makes refactoring very easy as you know exactly where your changes will have an effect
4. **It is perfect for React Native:** The ultimate goal of this project is having mobile apps and one of the easiest way to achieve that is with React Native. But react native does not use normal css. Using styled components is one of the easiest ways to style React Native components. This means that we could reuse components later with very little modification